### ISSUE TITLE
Suggested new feature to batch change permission, owner, group with recursive

### ISSUE THREAD

**Issue by Francewhoa** [doublecmd/doublecmd#593]
### User Story

As a user of **doublecmd**, I need to batch change both the owner and the group for a folder and all its sub-folders and sub-files. In other words, recursive. 

### Assumptions

* Assumes that by "**batch**" I mean all at once batch change both the owner and the group for a folder and all its sub-folders and sub-files. In other words, recursive
* Assumes that I searched both the documentation and the ticket/issues. But found no answer. I found how to change attributes under "**File**" ---> "**Change attribute**". There is a useful "**Recursive**" option in this window. But this changes only the attributes with recursive. Not the owner and group with recursive. I also found "**File**" ---> "**Show files properties**"---> "**Properties**". But unfortunately, this windows does not have a "**Recursive**" option.

### Suggested Resolution

How about using GNOME Commander as inspiration? Using GNOME Commander, those recursive actions can be done as follows:

**Batch Change Permissions**

1. Under "**File**" menu ---> "**Edit permission**" ---> This "**Set Permissions**" window opens. In this screenshot below, the number 3 shows the "**Apply recursively on**" / "**Appliquer récursivement pour**" option.
![batch_edit_permission---gnome_commander---screenshot---francewhoa---ksnip---2022-09-18---165936](https://user-images.githubusercontent.com/119942/190928037-5a2e22fc-c367-4976-9533-6644873707e1.png)
1. Using this "**Set Permissions**" / "**Permissions d'accès**" window, set permissions with **Recursive**
1. Click on the **"Validate"** button

**Batch Change Owner And Group**
1. Under "**File**" menu ---> "**Edit owner/group**" ---> This "**Chown**" window opens. In this screenshot below, the number 5 shows the "**Apply recursively on**" / "**Appliquer récursivement pour**" option.
   ![batch_edit_owner_and_group---gnome_commander---screenshot---francewhoa---ksnip---2022-09-18---170618](https://user-images.githubusercontent.com/119942/190928361-17289e1a-ffeb-40e7-8d84-15f02fa511f6.png)
1. Using this "**Chown**" window, set the appropriate Owner and Group with **Recursive**
1. Click on the **"Validate"** button

### Any Volunteer?

Any volunteer for a patch? Both me and the Ubertus.org team would be happy, as volunteer, to contribute testing and documentation, if needed.

**Comment by alexx2000** [comment]
I don't like how it's done in GNOME Commander. But I will redesign file properties window, make it like in other file managers and add **Recursive** checkbox.

**Comment by Francewhoa** [comment]
Hello @alexx2000 :) This is to say thank you for this new feature. Today I updated my DoubleCmd from your repository. Then noticed the new **Recursive** feature. Tried it. It works. Very easy to use. This is great.

Sorry for the very late confirmation. I needed to resolve health challenges. Feeling much better now. 

For those not familiar with this new feature, this screenshot below shows its location using "Files" menu ---> "Show file properties" option.

![screenshot---recursive---doublecmd---francewhoa---ksnip---2024-01-11---011300](https://github.com/doublecmd/doublecmd/assets/119942/ede2c5db-33b4-49d7-9e1a-b71bdab432b0)
